cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -50,12 +50,12 @@ This channel deploys the current release candidate for the `stable` channel. The
 
 #### stable release channel
```

Comment:
This channel deploys the latest release of TinyMCE that has passed our quality assurance process. The current version of TinyMCE available through the /5/ stable channel [can be found here](https://cloud.tinymce.com/5/version.txt).  The TinyMCE 5 stable channel can be loaded from this url [https://cloud.tinymce.com/5/tinymce.min.js](https://cloud.tinymce.com/5/tinymce.min.js).


<replace section with update>